scripts/export-triposg-onnx.py

```
@@ -0,0 +1,597 @@
+                args.opset)
+
+    # ---- int8 tier of the heaviest graph (the ~5.7 GB DiT) --------------------
+    # MatMul-only dynamic QInt8 (repo precedent from export-triposr-onnx.py:
+    # quantizing Conv would emit ConvInteger which our ORT CPU EP lacks; the
+    # DiT is MatMul-dominated so it still shrinks ~4x to a single-file model).
+    if not args.no_quant:
+        try:
+            from onnxruntime.quantization import quantize_dynamic, QuantType
+            int8_path = os.path.join(args.out, "triposg_dit_step_int8.onnx")
+            # per_channel is ESSENTIAL here: per-tensor dynamic quant on this
+            # DiT compounds over the flow loop (2 CFG calls/step, guidance 7×)
```

Ruff (0.15.20)

[warning] Comment contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF003)